packages/experimental/Text/src/Text.tsx

```diff
@@ -50,6 +51,15 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
     ? 'right'
     : align;
 
+  const onAccTap =
+    onAccessibilityTap ??
+    React.useCallback(
+      (event?) => {
+        props.onPress(event);
+      },
+      [props.onPress],
+    );
+
```

Comment:
Thanks for doing this! This feels like something `...pressableProps` should handle. I wonder if that's a valid upstream change in the future 🤔

Reply:
We should probably log that somewhere..... so for now I'll link this issue: https://github.com/microsoft/fluentui-react-native/issues/743

Reply:
[Task: 6276444](https://office.visualstudio.com/OC/_sprints/taskboard/CXE%20React/OC/CY22Q3%20(Jul-Sep)/2207-Jul/Jul02?workitem=5525048)

Created task under V1 Pressable

Reply:
We have a V1 Pressable task? :o